src/YRichText.h

```diff
@@ -206,6 +206,12 @@ class YRichText : public YWidget
      */
     virtual void setHScrollValue( const std::string & newValue );
 
+    /**
+     * Derived classes should implement this, method is used to trigger event
+     * like user has clicked link in the RichText
+     **/
+    virtual void clickLink( const std::string & url ) = 0;
```

Comment:
Since you can't click in NCurses, this should probably rather be `actvateLink()`. That would also be consistent with the other similar methods.

Reply:
+1. Renamed.